aws/diff_suppress_funcs.go

```diff
@@ -125,6 +125,8 @@ func suppressCloudFormationTemplateBodyDiffs(k, old, new string, d *schema.Resou
 	return normalizedOld == normalizedNew
 }
 
+var suppressOpenAPISpecificationDiffs = suppressCloudFormationTemplateBodyDiffs
```

Comment:
Probably best for long term maintainability is to rename `suppressCloudFormationTemplateBodyDiffs` as `suppressEquivalentJsonOrYamlDiffs` and use that.